Corgibytes.Freshli.Cli/CommandRunners/Cache/PrepareCacheActivity.cs

```diff
@@ -10,14 +10,17 @@ namespace Corgibytes.Freshli.Cli.CommandRunners.Cache;
 public class PrepareCacheActivity : IApplicationActivity
 {
     public PrepareCacheActivity(string cacheDirectory, string repositoryUrl = "", string? repositoryBranch = null,
-        string historyInterval = "")
+        string historyInterval = "", string gitPath = "")
     {
         CacheDirectory = cacheDirectory;
         RepositoryUrl = repositoryUrl;
         RepositoryBranch = repositoryBranch;
         HistoryInterval = historyInterval;
+        GitPath = gitPath;
     }
 
+    public string GitPath { get; init; }
```

Comment:
Storing the path to get in the cache database could potentially cause problems. It would be possible (however unlikely) for someone to change the path of the git executable between runs of the `freshli` application. This would mean that the git executable no longer exists in the same location as it did before.

If this is a piece of data that needs to be passed around, perhaps it needs to exist in something like an `ISettings` or `IApplicationSettings` interface, and that could be passed around instead.